package.json

```diff
@@ -9,7 +9,12 @@
   },
   "dependencies": {
     "@date-io/date-fns": "^1.3.13",
```

Comment:
I think we should remove "^"  from the versions because it is bug-prone.

Reply:
It upgrades the patch versions not major ones, why is it a bug-prone?

Reply:
Because we are not sure that our trusted libraries will not make major changes with the patch version.

Reply:
If team is ok with only using static versions(without "^", "~"), i'm also ok with that.

Reply:
It's OK to use static versions